Dead end detected: `musl` based glibc affects nodejs installation 
```bash
[INFO] --- frontend:1.12.1:install-node-and-npm (install node and npm) @ customer-space-portlet ---
[INFO] Installing node version v16.0.0
[INFO] Downloading https://cdn.npmmirror.com/binaries/node/v16.0.0/node-v16.0.0-linux-x64-musl.tar.gz to /home/ciagent/.m2/repository/com/github/eirslett/node/16.0.0/node-16.0.0-linux-x64-musl.tar.gz
[INFO] No proxies configured
[INFO] No proxy was configured, downloading directly
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.1:install-node-and-npm (install node and npm) on project customer-space-portlet: Could not download Node.js: Got error code 404 from the server. -> [Help 1]
```
Related plugin issue: https://github.com/eirslett/frontend-maven-plugin/issues/965

--- a/README.md
+++ b/README.md
@@ -53,7 +53,7 @@ It contains **Dockerfiles, Tests and configuration** for the following CI Images
 | exoplatform/ci:jdk17-mavend08       |   17.0.7    | 2.7 & 3  | 4.8 | Mavend 0.8.2   | [goss.yaml](mavend/jdk17-mavend08/tests/goss.yaml)
 | exoplatform/ci:jdk17-mavend09       |   17.0.7    | 2.7 & 3  | 4.8 | Mavend 0.9.0   | [goss.yaml](mavend/jdk17-mavend09/tests/goss.yaml)
 | exoplatform/ci:jdk17-maven39        |   17.0.7    | 2.7 & 3  | 4.8 | Maven 3.9.2    | [goss.yaml](maven/jdk17-maven39/tests/goss.yaml)
-| exoplatform/ci:jdk17-maven39-alpine |   17.0.7    | 2.7 & 3  | 4.8 | Maven 3.9.2    | [goss.yaml](maven/jdk17-maven39/tests/goss.yaml)
+| exoplatform/ci:jdk17-maven39-alpine |   17.0.7    | 2.7 & 3  | 4.8 | Maven 3.9.2    | [goss.yaml](maven/jdk17-maven39-alpine/tests/goss.yaml)
 
 
 ## Overview

--- a/maven/jdk17-maven39-alpine/Dockerfile
+++ b/maven/jdk17-maven39-alpine/Dockerfile
@@ -29,6 +29,7 @@ RUN  chown ${EXO_CI_USER}:${EXO_GROUP} ${HOME}/.web3j
 
 # Custom configuration for Maven
 ENV M2_HOME=/usr/share/maven
+# Alpine (musl) Do not override nodejs download URL; tl;dr https://github.com/eirslett/frontend-maven-plugin/issues/965
 ENV MAVEN_OPTS="-Dmaven.repo.local=${HOME}/.m2/repository -Xms1G -Xmx2G -XX:MaxMetaspaceSize=1G -Dcom.sun.media.jai.disableMediaLib=true -Djava.io.tmpdir=${EXO_CI_TMP_DIR} -Dmaven.artifact.threads=10 -Djava.awt.headless=true --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens java.base/java.nio.charset=ALL-UNNAMED --add-opens java.base/java.util.regex=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED -DnpmDownloadRoot=https://registry.npmmirror.com/npm/-/"
 ENV PATH=$JAVA_HOME/bin:$M2_HOME/bin:$PATH
 

--- a/maven/jdk17-maven39-alpine/Dockerfile
+++ b/maven/jdk17-maven39-alpine/Dockerfile
@@ -29,7 +29,7 @@ RUN  chown ${EXO_CI_USER}:${EXO_GROUP} ${HOME}/.web3j
 
 # Custom configuration for Maven
 ENV M2_HOME=/usr/share/maven
-ENV MAVEN_OPTS="-Dmaven.repo.local=${HOME}/.m2/repository -Xms1G -Xmx2G -XX:MaxMetaspaceSize=1G -Dcom.sun.media.jai.disableMediaLib=true -Djava.io.tmpdir=${EXO_CI_TMP_DIR} -Dmaven.artifact.threads=10 -Djava.awt.headless=true --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens java.base/java.nio.charset=ALL-UNNAMED --add-opens java.base/java.util.regex=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED -DnodeDownloadRoot=https://cdn.npmmirror.com/binaries/node/ -DnpmDownloadRoot=https://registry.npmmirror.com/npm/-/"
+ENV MAVEN_OPTS="-Dmaven.repo.local=${HOME}/.m2/repository -Xms1G -Xmx2G -XX:MaxMetaspaceSize=1G -Dcom.sun.media.jai.disableMediaLib=true -Djava.io.tmpdir=${EXO_CI_TMP_DIR} -Dmaven.artifact.threads=10 -Djava.awt.headless=true --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens java.base/java.nio.charset=ALL-UNNAMED --add-opens java.base/java.util.regex=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED -DnpmDownloadRoot=https://registry.npmmirror.com/npm/-/"
 ENV PATH=$JAVA_HOME/bin:$M2_HOME/bin:$PATH
 
 # Create needed directories for Maven & git

